en/guide/commands.md

```diff
@@ -49,18 +50,19 @@ npm run dev
 
 ## Production Deployment
 
-Nuxt.js lets your choose between 3 modes to deploy your application: Server Rendered, SPA or Static Generated.
+Nuxt.js lets your choose between three modes to deploy your application: Server Rendered, SPA or Static Generated.
 
 ### Server Rendered Deployment (Universal)
 
-To deploy, instead of running nuxt, you probably want to build ahead of time. Therefore, building and starting are separate commands:
+To deploy, instead of running `nuxt`, you probably want to build ahead of time. Therefore, building and starting are separate commands:
```

Comment:
If we speak about the command, add `. If we speak about the product, use « Nuxt » instead. I choose a command in this PR.